### PR TITLE
feat: dashboards section navigation flow

### PR DESCRIPTION
**Pull request by g10**
🔮 `feature branch`

### CODE REVIEW

File: centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,71 @@
+/* eslint-disable react-hooks/rules-of-hooks */
+import { useEffect, useState } from 'react';
+
+import { Meta, StoryObj } from '@storybook/react';
+
+// import { expect } from '@storybook/jest';
```
Comment (bdauria) [inline centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx:6]: Will we need that in the future?

File: centreon/packages/ui/style-dictionary.transform.ts [hunk 2]
```diff
@@ -0,0 +1,135 @@
...
+
+StyleDictionaryPackage.registerTransformGroup({
+  name: "styleguide",
+  transforms: ["attribute/cti", "name/cti/kebab", "size/px", "color/css"],
+})
+
+// Generate the base tokens
+
+// eslint-disable-next-line no-console
+console.log("\x1b[33m%s\x1b[0m", "\n\n⚙️ base")
```
Comment (bdauria) [inline centreon/packages/ui/style-dictionary.transform.ts:120]: are the console log used for debug purposes? if so we can get rid of them (applies to the one below as well)

**Review by bdauria — COMMENTED** [review]
LGTM if commented lines are helpful for future unfinished work

File: centreon/packages/ui/style-dictionary.transform.ts (hunk 2 above)
Comment (g10) [reply centreon/packages/ui/style-dictionary.transform.ts:120]: fyi. this transform is a cli util, the log is to structure the output
Screenshot 2023-06-27 at 11 18 16

File: centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx (hunk 1 above)
Comment (g10) [reply centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx]: this specific one not, been removed

in the `front_src` there are chunks of code that are temporarily commented out, they should be marked with a FIXME / TODO if it's intentional / for future unfinished work

**Review by bdauria — APPROVED** [review]

File: centreon/packages/ui/src/utils/useResizeObserver.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
+// https://github.com/ZeeCoder/use-resize-observer
+import useResizeObserver from 'use-resize-observer';
```
Comment (Noha-ElAbrouki) [inline centreon/packages/ui/src/utils/useResizeObserver.ts:2]: just 581 star :p

File: centreon/www/front_src/src/Dashboards/api/decoders.ts [hunk 4]
```diff
@@ -1,41 +1,87 @@
...
-    updatedBy: 'updated_by'
-  }
-);
-
-export const dashboardListDecoder = buildListingDecoder({
-  entityDecoder: dashboardDecoder,
-  entityDecoderName: 'Dashboard Listing',
-  listingDecoderName: 'Dashboards'
-});
+/* eslint-disable typescript-sort-keys/interface */
```
Comment (Noha-ElAbrouki) [inline centreon/www/front_src/src/Dashboards/api/decoders.ts:1]: it necessary 🤔

File: centreon/www/front_src/src/Dashboards/components/DashboardAccessRights/DashboardAccessRightsModal.tsx [hunk 5]
```diff
@@ -0,0 +1,56 @@
...
+  );
+
+  return (
+    <Modal open={isDialogOpen} onClose={closeDialog}>
+      <Modal.Header>{labels.modalTitle}</Modal.Header>
+      <Modal.Body />
+      <Modal.Actions
+        labels={labels.form.actions}
+        onCancel={closeDialog}
+        // onConfirm={submit}
```
Comment (Noha-ElAbrouki) [inline centreon/www/front_src/src/Dashboards/components/DashboardAccessRights/DashboardAccessRightsModal.tsx:50]: 🤔

File: centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx [hunk 6]
```diff
@@ -0,0 +1,71 @@
...
+  }
+};
+
+export default meta;
+type Story = StoryObj<typeof TextOverflowTooltip>;
+
+export const Default: Story = {
+  args: {
+    label:
+      'Qui deserunt pariatur quis. Duis nisi velit culpa labore ipsum reprehenderit sunt laborum anim sint quis magna consequat amet. Voluptate tempor nostrud eiusmod enim qui reprehenderit.'
```
[the remark this answers is not in the document]
Comment (g10) [reply centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx:22]: https://en.wikipedia.org/wiki/Lorem_ipsum

File: centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (g10) [reply centreon/packages/ui/src/components/Tooltip/TextOverflowTooltip/TextOverflowTooltip.stories.tsx]: removed already

File: centreon/www/front_src/src/Dashboards/api/decoders.ts (hunk 4 above)
Comment (g10) [reply centreon/www/front_src/src/Dashboards/api/decoders.ts:1]: preferred to have logical grouping then alphabetical for schema

File: centreon/www/front_src/src/Dashboards/components/DashboardAccessRights/DashboardAccessRightsModal.tsx (hunk 5 above)
Comment (g10) [reply centreon/www/front_src/src/Dashboards/components/DashboardAccessRights/DashboardAccessRightsModal.tsx:50]: yes, this modal is a placeholder, will be wired up in next PR (currently in other feature branch, therefore this branch needs to be merged first)

**Pull request merged**